### PR TITLE
Using Helm with Charmed Kubernetes

### PR DESCRIPTION
**Pull request by natalytvinova**
Tutorial for using Helm with Charmed Kubernetes.

### CODE REVIEW

File: tutorials/containers/using-helm-with-canonical-kubernetes/using-helm-with-canonical-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Duration: 2:00
+
+Make sure that your Kubernetes cluster is running and kubectl config is in ~/.kube/config.
+
+`juju scp kubernetes-master/0:config ~/.kube/config`
+
+Also make sure you have opened the Helm port 44134 on all your worker nodes:
+
+```
+juju run --unit kubernetes-worker/0 "open-port 44134"
```
Comment (timClicks) [inline tutorials/containers/using-helm-with-canonical-kubernetes/using-helm-with-canonical-kubernetes.md]: Very recently, `juju run` was replaced by `juju exec`. At the moment, they both work. You can ask Juju to exec a command across every unit of an application in a single line:

```
juju exec --app kubernetes-worker "open-port 44134"
```

**Review by timClicks — COMMENTED** [review]
This is a neat little tutorial @natalytvinova. I would change the title to "Deploying a Harm charm on Charmed Kubernetes" though. I originally thought that this would be more in-depth.

(Hi from the Juju team, by the way! :wave:)

**Comment by natalytvinova** [comment]
Thank you for the tips and suggestions @timClicks. I applied them, also added Helm 2 instructions as these versions are very different. I do want to add some more depth to the tutorial later.

**Comment by steverydz** [comment]
Thanks for submitting a PR for this tutorial. We have moved our tutorials to discourse. To contribute going forwards please go to https://discourse.ubuntu.com/c/tutorials.

**Pull request closed without merging**